### PR TITLE
📖 Simplify some selection criteria in quickstart

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates the existing quickstart to use simpler selection criteria available now. This PR also replaces some Spanish with English and adds a note about the upsync section of the exhibited EdgePlacement object.

## Related issue(s)

Fixes #

### CODE REVIEW

**Review by KPRoche — APPROVED** [review]
These changes look great to me. I was just reviewing the quickstart vs the user-quickstart today and noticed the discrepancy

File: docs/content/Getting-Started/quickstart-subs/quickstart-2-apache-example-deployment-d-create-and-deploy-apache-into-clusters.md [hunk 1]
```diff
@@ -21,17 +21,14 @@ spec:
   downsync:
   - apiGroup: ""
     resources: [ configmaps ]
-    namespaceSelectors:
-    - matchLabels: {"common":"si"}
```
[the remark this answers is not in the document]
Comment (clubanderson) [reply docs/content/Getting-Started/quickstart-subs/quickstart-2-apache-example-deployment-d-create-and-deploy-apache-into-clusters.md:25]: looks good

**Pull request merged**